bottlecap/src/proxy/interceptor.rs

```diff
@@ -172,6 +187,25 @@ async fn invocation_next_proxy(
         });
     }
 
+    // K9 / ASM
+    if let Some(appsec_processor) = appsec_processor {
+        if let Some(request_id) = intercepted_parts
+            .headers
+            .get("Lambda-Runtime-Aws-Request-Id")
```

Comment:
I remember either in next or response we can get this by checking at the Path, we can leverage Axum for it, will check again which one was it